tests/unit_tests/services/test_dds.py

```diff
@@ -386,27 +386,23 @@ def test_release_project_nomail(self):
             ])
 
     @gen_test
-    def test_get_dds_project_title(self):
-        mock_dds_project = [{
-                "Access": True,
-                "Last updated": "Fri, 01 Jul 2022 14:31:13 CEST",
-                "PI": "pi@email.com",
-                "Project ID": "snpseq00025",
-                "Size": 25856185058,
-                "Status": "In Progress",
-                "Title": "AB1234"
-                }]
+    def test_get_dds_version(self):
```

Comment:
perhaps a broader question, but this tests the code in the `models/DDSProject` class. for consistency, shouldn't the unit test be under `unit_tests/models` instead of `unit_tests/services`?

Reply:
I don't think you need to move it now though, it's probably fine where it is.

Reply:
Oooh, I now see the different folders in the tests are been used. Thank you for that. Well noted.